elliott/elliottlib/cli/snapshot_cli.py

```
@@ -167,38 +199,50 @@ async def get_pullspecs(pullspecs: list, image_repo_pull_secret: str):
+                    if isinstance(record, KonfluxBuildRecord):
+                        comp_name = KonfluxImageBuilder.get_component_name(app_name, record.name)
                         await self.konflux_client.get_component__caching(comp_name, strict=True)
-                else:
-                    # fbc component name is determined from the image it builds for, which is not stored in the DB
-                    # rather than hack something up, let it fail for now
-                    raise e
+                    elif isinstance(record, KonfluxBundleBuildRecord):
+                        comp_name = KonfluxOlmBundleBuilder.get_component_name(app_name, record.name)
+                        try:
+                            await self.konflux_client.get_component__caching(comp_name, strict=True)
+                        except Exception:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)